tensorflow_addons/utils/keras_utils.py

```diff
@@ -19,10 +19,50 @@
 
 import tensorflow as tf
 
-# TODO: find public API alternative to these
-from tensorflow.python.keras.utils import conv_utils  # pylint: disable=unused-import
-
 
 def register_keras_custom_object(cls):
     tf.keras.utils.get_custom_objects()[cls.__name__] = cls
     return cls
+
+
+def normalize_tuple(value, n, name):
+    """Transforms a single integer or iterable of integers into an integer
+    tuple. Ported from tensorflow.python.keras.util.
+
+    Arguments:
+      value: The value to validate and convert. Could an int, or any iterable
+        of ints.
+      n: The size of the tuple to be returned.
+      name: The name of the argument being validated, e.g. "strides" or
+        "kernel_size". This is only used to format error messages.
+
+    Returns:
+      A tuple of n integers.
+
+    Raises:
+      ValueError: If something else than an int/long or iterable thereof was
+        passed.
+    """
+    if isinstance(value, int):
+        return (value,) * n
+    else:
+        try:
+            value_tuple = tuple(value)
+        except TypeError:
+            raise ValueError('The `' + name +
+                             '` argument must be a tuple of ' + str(n) +
+                             ' integers. Received: ' + str(value))
+        if len(value_tuple) != n:
+            raise ValueError('The `' + name +
+                             '` argument must be a tuple of ' + str(n) +
+                             ' integers. Received: ' + str(value))
+        for single_value in value_tuple:
+            try:
+                int(single_value)
+            except (ValueError, TypeError):
+                raise ValueError('The `' + name +
+                                 '` argument must be a tuple of ' + str(n) +
+                                 ' integers. Received: ' + str(value) + ' '
+                                 'including element ' + str(single_value) +
+                                 ' of type' + ' ' + str(type(single_value)))
+        return value_tuple
```

Comment:
So I noticed that the indentation of the `return` statement seems to have drifted one unit to the right.  It doesn't matter semantically but I suppose the intent is a bit clearer with the new approach---because the return statement lives in the conditional?

Reply:
Interesting... it differs from keras and tensorflow:
https://github.com/keras-team/keras/blob/master/keras/utils/conv_utils.py
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/utils/conv_utils.py

Quite hard to notice on the TF version (2 spaces...), but as you said it behaves the same.

Reply:
keras Version looks weird, I think both coding styles below are acceptable:

```python
if condition:
    return value
return default_value

# or
if condition_a:
    return value_a
else:
    return value_b
```